src/Serilog/Enrichers/EnvironmentUserNameEnricher.cs

```diff
@@ -21,7 +21,11 @@
 namespace Serilog.Enrichers
 {
     /// <summary>
+#if USERNAMEFROMENV
```

Comment:
I'd prefer not including changes to this enricher in this PR - let's keep it to `LogContext` and leave the enricher work for a later PR (there's some factoring work being done that will affect this).